src/Ng/Jobs/DB2CatalogJob.cs

```diff
@@ -303,8 +289,7 @@ private static Uri MakeLastEditedUri(string source, DateTime since, int top)
                 if (!string.IsNullOrEmpty(auditEntry.PackageId) && !string.IsNullOrEmpty(auditEntry.PackageVersion) && auditEntry.TimestampUtc > since)
                 {
                     // Mark the package "deleted"
-                    IList<FeedPackageIdentity> packages;
-                    if (!result.TryGetValue(auditEntry.TimestampUtc.Value, out packages))
+                    if (!result.TryGetValue(auditEntry.TimestampUtc.Value, out IList<FeedPackageIdentity> packages))
```

Comment:
super nit: you can do `out var packages`

Reply:
Fixed in https://github.com/NuGet/NuGet.Services.Metadata/pull/524/commits/81d386ceb95e2f5839d6e63029ee23ca1b22326b